projects/batfish-common-protocol/src/test/java/org/batfish/common/ContainerTest.java

```diff
@@ -1,39 +0,0 @@
-package org.batfish.common;
```

Comment:
It should be a rename, I moved Container class into org.batfish.common.pojo. I don't know why it shows as a deletion and a creation of the class.